composer.json

```diff
@@ -19,6 +19,9 @@
         "phpunit/phpunit": "^9.5",
         "squizlabs/php_codesniffer": "^3"
     },
+    "conflict": {
+        "silverstripe/auditor": "<2.4.0"
```

Comment:
Why are you adding this conflict? Is there a failed CI run you can point at for example where that specific version was causing issues?

Reply:
prefer lowest issue
```
1) CWP\CWP\Tests\Extensions\WorkflowDefinitionExtensionTest::testCreateDefaultWorkflowTest
implode(): Passing glue string after array is deprecated. Swap the parameters
/home/runner/work/cwp/cwp/vendor/silverstripe/auditor/code/AuditHook.php:132
```
https://github.com/creative-commoners/cwp/runs/7332407622?check_suite_focus=true#step:12:68

Reply:
That was resolved in 2.2.1. Please update to `<2.2.1`
https://github.com/silverstripe/silverstripe-auditor/releases/tag/2.2.1

Reply:
updated